ocaml/sdk-gen/java/autogen/xen-api/pom.xml

```diff
@@ -109,7 +109,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
-                <version>3.2.0</version>
+                <version>3.6.3</version>
```

Comment:
Belatedly I realised that upgrade to a version >=3.3.0 comes with a breaking change:

if the goal `jar` is specified, building multiple phases (like `mvn package install` that we do in the pipeline) causes the build to fail with `Presumably you have configured maven-source-plugn to execute twice times in your build. You have to configure a classifier for at least on of them` (the errors in the message are all real...)

The solution is either to use goal `jar-no-fork` or avoid specifying mutliple phases (for example, in the pipeline we don't need the install bit anymore since we don't build the samples within the same build, so it's probably easier to correct it there since this PR is merged now).